queries/v/textobjects.scm

```diff
@@ -0,0 +1,71 @@
+;; assignment
+[(short_var_declaration
+    left: (_) @assignment.lhs
+    right: (_)* @assignment.rhs)
+  (assignment_statement
+    left: (_) @assignment.lhs
+    right: (_)* @assignment.rhs)]
+
+[(short_var_declaration
+    left: (_) @assignment.inner)
+  (assignment_statement
+    left: (_) @assignment.inner)]
+
+[(short_var_declaration
+    right: (_) @assignment.inner)
+  (assignment_statement
+    right: (_) @assignment.inner)]
+
+;; block
+(_ (block) @block.inner) @block.outer
+
+;; call
+(call_expression) @call.outer
+(call_expression
+  arguments: (argument_list . "(" . (_) @_start (_)? @_end . ")"
+  (#make-range! "call.inner" @_start @_end)))
+
+;; class: structs
+(struct_declaration
+  (struct_field_declaration_list . "{" . (_) @_start @_end (_)? @_end . "}"
+  (#make-range! "class.inner" @_start @_end)))
+
+(struct_declaration) @class.outer
+
+;; comment
+((comment) @comment.inner @comment.outer
+  (#offset! @comment.inner 0 3 0))
```

Comment:
Is `comment.inner` guaranteed to have offset of 3? Maybe it's fine for now, but to be more accurate maybe we need to offset 2 and delect the leading spaces.

Reply:
I've just created a PR for `comment.inner` for python to do exactly this. Will push the fix for `V` now.

Reply:
I've just created a PR for `comment.inner` for python to do exactly this. Will push the fix for `V` now.